pegasus/forms/hoc_signup_2017.rb

```diff
@@ -30,6 +30,21 @@ def self.normalize(data)
     result
   end
 
+  def self.process(data)
+    # If there is an nces school id then we need to load that school's address and use it as the event location.
+    # A school id of "-1" indicates that the school was not found in the school dropdown so we treat that the
+    # same as no school id.
+    nces_school_id = data['nces_school_s']
+    if nces_school_id && (nces_school_id != "-1")
```

Comment:
nit: parens necessary?  haven't seen this pattern in our ruby before..

Reply:
I removed them. My Java brain really wants to put parens around the whole if condition.